src/System.Diagnostics.Debug/tests/DebuggerBrowsableAttributeTests.cs

```diff
@@ -1,4 +1,5 @@
-﻿// The .NET Foundation licenses this file to you under the MIT license.
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This, and a couple of others with identical messages, is the one case I changed an existing header that wasn't just a slight error. It looks like the first line is omitted by mistake, but maybe there was a reason it was deliberately left out.